tests/js/sentry-test/performance/initializePerformanceData.ts

```diff
@@ -175,3 +175,57 @@ export function generateSuspectSpansResponse(opts?: {
     return suspectSpans;
   });
 }
+
+export function generateSampleEvent(): EventTransaction {
+  const event = {
+    id: '2b658a829a21496b87fd1f14a61abf65',
+    eventID: '2b658a829a21496b87fd1f14a61abf65',
+    title: '/organizations/:orgId/discover/results/',
+    type: 'transaction',
+    startTimestamp: 1622079935.86141,
+    endTimestamp: 1622079940.032905,
```

Comment:
```suggestion
export function generateSampleEvent(event: Partial<EventTransaction>): EventTransaction {
  const event = {
    ...event, // but this should be at the end of the object
    id: '2b658a829a21496b87fd1f14a61abf65',
    eventID: '2b658a829a21496b87fd1f14a61abf65',
    title: '/organizations/:orgId/discover/results/',
    type: 'transaction',
    startTimestamp: 1622079935.86141,
    endTimestamp: 1622079940.032905,
```
But even better if u could update nested properties too.

Reply:
Probably going to delete this function actually since it's not really necessary now that we have the `TransactionEventBuilder`